db/polls.rb

```diff
@@ -0,0 +1,439 @@
+author = Administrator.first.user
+
+poll_description_en = <<~TEXT
+  <p>This participatory process to determine the theme of the "Prospective Research" programme for the
```

Comment:
Poll summary and description are rendered using `simple_format`, so using `<p>` here results in double paragraph tags and double closing paragraph tags, which is invalid HTML.

Note poll answers are rendered using `wysiwyg`, so in the case of the answers, the `<p>` tag is correct. So I guess it's enough if we change `poll_description_en`, `poll_description_fr`, `poll_summary_en` and `poll_summary_fr`.